py_bind/optv/parameters.pyx

```diff
@@ -1,20 +1,50 @@
-#Implementation of Python binding to parameters.h
+# Implementation of Python binding to parameters.h
 from libc.stdlib cimport malloc, free
+from libc.string cimport strncpy
+
 import numpy
 
+cdef extern from "optv/parameters.h":
+    int c_compare_mm_np "compare_mm_np"(mm_np * mm_np1, mm_np * mm_np2)
+    
+    track_par * c_read_track_par "read_track_par"(char * file_name)
+    int c_compare_track_par "compare_track_par"(track_par * t1, track_par * t2)
+    
+    sequence_par * c_read_sequence_par "read_sequence_par"(char * filename)
+    sequence_par * c_new_sequence_par "new_sequence_par"()
+    void c_free_sequence_par "free_sequence_par"(sequence_par * sp)
+    int c_compare_sequence_par "compare_sequence_par"(sequence_par * sp1, sequence_par * sp2)
+    
+    volume_par * c_read_volume_par "read_volume_par"(char * filename);
+    int c_compare_volume_par "compare_volume_par"(volume_par * v1, volume_par * v2);
+    
+    control_par * c_read_control_par "read_control_par"(char * filename);
+    control_par * c_new_control_par "new_control_par"(int cams);
+    void c_free_control_par "free_control_par"(control_par * cp);
+    int c_compare_control_par "compare_control_par"(control_par * c1, control_par * c2);
+
 cdef class MultimediaParams:
 
     def __init__(self, **kwargs):
         
-        self._mm_np = <mm_np *>malloc(sizeof(mm_np))
-        
-        self.set_nlay(kwargs['nlay'])
-        self.set_n1(kwargs['n1'])
-        self.set_n2(kwargs['n2'])
-        self.set_d(kwargs['d'])
-        self.set_n3(kwargs['n3'])
-        self.set_lut(kwargs['lut'])
-    
+        self._mm_np = < mm_np *> malloc(sizeof(mm_np))
+        if kwargs.has_key('nlay'): 
+            self.set_nlay(kwargs['nlay'])
+        if kwargs.has_key('n1'):
+            self.set_n1(kwargs['n1'])
+        if kwargs.has_key('n2'):
+            self.set_n2(kwargs['n2'])
+        if kwargs.has_key('d'):
+            self.set_d(kwargs['d'])
+        if kwargs.has_key('n3'):
+            self.set_n3(kwargs['n3'])
+        if kwargs.has_key('lut'):
```

Comment:
Alex just found a bug that requires a change here: If there is no ``lut``, lut should be set to 0. Possible code:

```
kwargs.setdefault('lut', 0)
self.set_lut(kwargs['lut'])
```